src/systemd/sd-dhcp-client.h

```diff
@@ -149,6 +150,9 @@ int sd_dhcp_client_set_hostname(
 int sd_dhcp_client_set_vendor_class_identifier(
                 sd_dhcp_client *client,
                 const char *vci);
+int sd_dhcp_client_set_user_class(
+                sd_dhcp_client *client,
+                const char *user_class);
```

Comment:
user_class → user_classes.

Public API should take a vector of opaque strings. Please convert this to take an array of `struct iovec`.

Reply:
I guess you mean 
```
sd_dhcp_client_set_user_class(
+                sd_dhcp_client *client,
+                struct iovec iov[]);
```

That will break `dhcp_option_append`  and option_append etc ..

Reply:
dhcp_option_append is an internal function. You might need to update it.

Reply:
`option_append`  is filling of the buffer(option / data) that will goto iovec.